packages/openapi-generator/src/document-converter.ts

```diff
@@ -15,15 +15,15 @@ const { readFile } = promises;
 
 /**
  * Convert an OpenAPI document to ensure smooth parsing and generation thereafter.
- * Documents are expected to be formatted as JSON, OpenAPI version 3 and only have one "default" tag.
+ * Documents are expected to be formatted as JSON and compliant with OpenAPI version 3.
  * @param filePath File content of the original spec.
  */
 export async function convertOpenApiSpec(
   filePath: string
 ): Promise<OpenAPIV3.Document> {
   const file = await parseFileAsJson(filePath);
   const openApiDocument = await convertDocToOpenApiV3(file);
-  return convertDocToGlobalTag(convertDocToUniqueOperationIds(openApiDocument));
+  return convertDocToUniqueOperationIds(openApiDocument);
```

Comment:
Luckily you didn't delete the `convertDocToGlobalTag` method yet. I think it would be a good idea, to add the 'default' tag, where there are no tags or only an empty list of tags.